src/components/Accordion.tsx

```diff
@@ -125,9 +127,11 @@ const _Accordion = (
           }}
         >
           <CenteredRow gridGap={2}>
-            <Text variant="medium" fontWeight={2} color="blue">
+            <AccordionTitle variant="medium" fontWeight={2} color="blue">
               {title}
-              <div style={subtitleStyle}>{type ? ' - ' + type : ''}</div>
+            </AccordionTitle>
+            <Text variant="medium" fontWeight={4} color="grey">
```

Comment:
Our `colors` object in src/theme.ts uses `gray` instead of `grey`, so I don't think this is actually using our theme colours.
```suggestion
            <Text variant="medium" fontWeight={4} color="gray">
```

Reply:
hmm just realized that the `gray` is darked than the `grey`, but honestly both work!